drivers/gps/gps_sim/gps_sim.c

```diff
@@ -0,0 +1,393 @@
+/*
```

Comment:
This file was supposed to be moved not copied. Same goes for `nrf9160_gps.c`.

Reply:
Thanks, that was a mistake